statefun-e2e-tests/statefun-smoke-e2e-java/src/main/java/org/apache/flink/statefun/e2e/smoke/generated/Command.java

```diff
@@ -0,0 +1,5005 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: commands.proto
+
+package org.apache.flink.statefun.e2e.smoke.generated;
```

Comment:
Now that the `SimpleVerificationServer` no longer exposes `TypedValue` on the surface, I believe you no longer need to pre-generate these Protobuf messages, and can just use the Maven protoc plugin to do that for you.

Reply:
I'm not quite get this one. 
For driver(previously common), we do not generate messages beforehand. 
It was my dirty work which generates messages with shaded protobuf. With your suggestion to remove TypedValueUtil, those pre-generated messages are gone in driver. 
Or you are saying to remove generated messages in smoke-e2e-java?

Reply:
OK. I looked into the code and kind of get it. Since no TypedValue is used in junit runtime, there's no need to resolve the shaded/non-shaded issue.